util/ipfs.js

```diff
@@ -0,0 +1,20 @@
+import Bottleneck from 'bottleneck'
+
+const gatewayUrl = process.env.IPFS_GATEWAY_URL
+
+const rateLimit = parseInt(process.env.IPFS_GATEWAY_RATE_LIMIT, 10)
+
+const gateway = new Bottleneck({
```

Comment:
WOOOW!

Reply:
Wonder if this applies to a paid/pro gateway. I think we go around that on other projects using a paid gateway.